apps/framework-cli/src/framework/python/wrappers/consumption_runner.py

```diff
@@ -122,16 +123,64 @@ def query(self, input, variables):
                 params[variable_name] = f'{{p{i}: {t}}}'
                 values[f'p{i}'] = value
         clickhouse_query = input.format_map(params)
-       
-        # We are not using the result of the ping
-        # but this ensures that if the clickhouse cloud service is idle, we 
-        # wake it up, before we send the query.
-        self.ch_client.ping()
 
         val = self.ch_client.query(clickhouse_query, values)
 
         return list(val.named_results())
 
+class WorkflowClient:
+    def execute(self, name: str, input_data: Any) -> None:
+        moose_cli = "/usr/local/bin/moose-cli"
```

Comment:
Not quite sure what should be. It's called `moose` in prod container, and `moose-cli` locally 🤔

Reply:
🤔 I was asking myself the same thing. It is kinda what do we want to standardize on, the code the send a request to the temporal server through their lib seem pretty simple. 

Also not having a dependency on the cli means we can use serverless envs eventually to deploy the backend.

Reply:
I tried the network call approach. It worked and doesn't look too bad. I made the temporal client initialized once when the http server comes up.